tests/test-th-land-patterns.stanza

```diff
@@ -0,0 +1,66 @@
+#use-added-syntax(jitx)
+defpackage ex:
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import ocdb/land-patterns
+  import ocdb/defaults
+  import ocdb/tolerance
+  import ocdb/box-symbol
+
+set-default-rules(default-rules)
+pcb-component radial (r:String, p?:True|False):
+  reference-prefix = r
+  pcb-landpattern lp:
+    make-two-pin-radial-landpattern(
+      tol(5.0, 0.5),
+      2.0,
+      tol(0.5, 0.0)
+      p?
+    )
+  if p?:
+    pin a
+    pin c
+    landpattern = lp(a => lp.a, c => lp.c)
+  else:
+    port p: pin[[1 2]]
+    landpattern = lp(p[1] => lp.p[1], p[2] => lp.p[2])
+  make-box-symbol()
+pcb-component axial (r:String, p?:True|False):
```

Comment:
what is representing `p?`

Reply:
r is the reference designator and p? Is if it is polarized. I can rename, but these are not public facing